library/src/com/google/maps/android/geojson/GeoJsonLineStringStyle.java

```diff
@@ -0,0 +1,168 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import java.util.Observable;
+
+/**
+ * A class that allows for GeoJsonLineString objects to be styled and for these styles to be
+ * translated into a PolylineOptions object
+ */
+public class GeoJsonLineStringStyle extends Observable implements GeoJsonStyle {
+
+    private final static String GEOMETRY_TYPE_REGEX
+            = "LineString|MultiLineString|GeometryCollection";
+
+    private final PolylineOptions mPolylineOptions;
+
+    /**
+     * Creates a new LineStringStyle object
+     */
+    public GeoJsonLineStringStyle() {
+        mPolylineOptions = new PolylineOptions();
+    }
+
+    /**
+     * Gets the type of geometries this style can be applied to
+     *
+     * @return type of geometries this style can be applied to
+     */
+    @Override
+    public String getGeometryType() {
+        return GEOMETRY_TYPE_REGEX;
+    }
+
+    /**
+     * Gets the color of the GeoJsonLineString
+     *
+     * @return color of the GeoJsonLineString
+     */
+    public int getColor() {
+        return mPolylineOptions.getColor();
+    }
+
+    /**
+     * Sets the color of the GeoJsonLineString
```

Comment:
What does the integer mean? Is it an RGB formatted like 0xRRGGBB? RGBA? Is it a resource ID?